docs/data/installation/docker-compose.md

```diff
@@ -19,9 +19,14 @@ Next create a .env-file use setting for your installation:
 
 ```bash
 RACK_ENV=production
-AHA_SECRET_PERMITTED_ORIGINS=http://localhost
+AHA_SECRET_BASE_URL=https://yourdomain.com
+AHA_SECRET_PERMITTED_ORIGINS=https://yourdomain.com
```

Comment:
I think that http://localhost would make more sense, since it works at least locally out of the box